resource_container/ResourceContainer.py

```diff
@@ -334,7 +334,12 @@ def identifier(self):
         if 'identifier' in self.resource and self.resource['identifier']:
             return self.resource['identifier']
         if 'id' in self.resource and self.resource['id']:
-            return self.resource['id']
+            id = self.resource['id']
```

Comment:
My only concern would be if this repo had a v0.2 manifest.yaml file the first if would be what passes. I'm actually thinking we should just add those resource types to the possibe Bible types, here https://github.com/unfoldingWord-dev/tx-manager-lambda/blob/develop/functions/convert_usfm2html/module.json#L5 and here https://github.com/unfoldingWord-dev/tx-manager/blob/develop/converters/usfm2html_converter.py#L15